test/routes/sent-resubmit.test.js

```diff
@@ -0,0 +1,69 @@
+const request = require('supertest')
+const {
+  createLicenceServiceStub,
+  createPrisonerServiceStub,
+  appSetup,
+  auditStub,
+  createSignInServiceStub,
+} = require('../supertestSetup')
+
+const standardRouter = require('../../server/routes/routeWorkers/standardRouter')
+const createRoute = require('../../server/routes/sent')
+
+describe('sent', () => {
```

Comment:
there is already a sent test that covers the other examples - could this not go there?

Reply:
same with the `send` file?

Reply:
done for sent
done for send